pages/platform/integrations/prefect_tuto_02_block_automation/guide.en-gb.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Prefect - Tutorial - Emails notification with blocks and automations
+slug: prefect/email-notification
+excerpt: Discover how to send email notification through blocks and automations in Prefect Cloud
+section: Prefect
+order: 02
+updated: 2023-05-12
+---
+
+**Last updated 12th May 2023**
+ 
+## Objective
+
+The purpose of this tutorial is to show you how to send a notification (in this case an email) with Prefect Cloud. This result will be achieved through two Prefect primitives: blocks and automations. At the end of the tutorial, you will be able to send an email via Prefect. You will also be able to understand the concept of blocks and automations and how to use them.  
+ 
+## Requirements
+
+- Basic Knowledge or Prefect.io
+- First configurationf for Prefect and OVHcloud. See our guide [Getting started](/pages/platform/integrations/prefect_guide_01_getting_started)
+- An email address
+
+## Instructions
+ 
+### Configure an Email block
+
+In Prefect, "blocks" are pre-built, reusable building blocks that encapsulate common tasks or operations. They are designed to make it easier for users to build workflows by abstracting away the underlying complexity and allowing them to focus on the high-level logic of their pipelines. Blocks can be thought of as building blocks that can be pieced together to create complex workflows. They are created and maintained by the Prefect community and can be easily shared and used by other users.
+
+To access the lists of blocks, go on your Prefect Cloud account. In the sidebar, you can see the tab `Blocks`. Click on it and you will be able to see a list of software. It can be Docker, Github, json... Here, we will use the Email Block. It should be similar to this: 
+
+![image](images/email_block.png){.thumbnail}
+
+Click on `Add +`button and fill the block with a name and the desired email address, where you will receive a notification. You can enter multiple email address. Once your block is created, let's see what is an automation and why it will be useful to send our email. 
```

Comment:
```suggestion
Click on `Add +`button and fill the block with a name and the desired email address where you will receive a notification. You can enter multiple email address. Once your block is created, let's see what is an automation and why it will be useful to send our email. 
```